theano/gof/op.py

```diff
@@ -694,6 +694,9 @@ def __ne__(self, other):
     # Python implementation #
     #########################
 
+    def grad2(self, inputs, outputs, output_grads):
```

Comment:
Otherwise, we could use `Lop` for the name. This is less easily understandable, but technically more correct than "grad".